go/contacts/resolution_notice.go

```diff
@@ -0,0 +1,69 @@
+package contacts
+
+import (
+	"context"
+
+	"github.com/keybase/client/go/encrypteddb"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+type ContactResolution struct {
+	Description  string
+	ResolvedUser keybase1.User
+}
+
+func getKeyFn(mctx libkb.MetaContext) func(ctx context.Context) ([32]byte, error) {
+	keyFn := func(ctx context.Context) ([32]byte, error) {
+		return encrypteddb.GetSecretBoxKey(ctx, mctx.G(),
```

Comment:
this is device key right? i forget if we wanted to work on all devices or just one

Reply:
(if we do, there should be a test for multi device)